docs/contributing.rst

```diff
@@ -146,6 +146,19 @@ are not covered by tests.
 
 The test suite is run automatically by CI.
 
+Test files live in ``tests`` directory, test filename naming convention:
```

Comment:
They're in the ``sgkit/tests`` directory, right?

Reply:
Fixed.